addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -0,0 +1,140 @@
+thing-type.config.netatmo.bridge.username.description = Benutzer zur Authentifizierung an der Netatmo API.
+
+thing-type.config.netatmo.bridge.password.label = Passwort
+thing-type.config.netatmo.bridge.password.description = Passwort zur Authentifizierung an der Netatmo API.
+
+thing-type.config.netatmo.bridge.readStation.label = Wetterstation API
+thing-type.config.netatmo.bridge.readStation.description = Aktiviert den Zugriff auf die Wetterstation API (lesend).
+
+thing-type.config.netatmo.bridge.readThermostat.label = Thermostat API
+thing-type.config.netatmo.bridge.readThermostat.description = Aktiviert den Zugriff auf die Thermostat API (lesend und schreibend).
+
+# thing types
+thing-type.netatmo.NAMain.label = Haupt-Indoor-Modul
+thing-type.netatmo.NAMain.description = Haupt-Indoor-Modul, liefert Daten, wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit, CO2 Gehalt und Lautst�rke.
+
+thing-type.netatmo.NAModule1.label = Outdoor-Modul
+thing-type.netatmo.NAModule1.description = Outdoor-Modul, liefert Daten, wie z.B. Temperatur und Luftdruck.
+
+thing-type.netatmo.NAModule2.label = Windmesser
+thing-type.netatmo.NAModule2.description = Windmesser, liefert Daten, wie z.B. Windrichtung und Windst�rke.
+
+thing-type.netatmo.NAModule3.label = Regenmesser
+thing-type.netatmo.NAModule3.description = Regenmesser, liefert Daten, wie z.B. Echtzeit-Messungen zur Intensit�t des Regens.
+
+thing-type.netatmo.NAModule4.label = Zusatz-Indoor-Modul
+thing-type.netatmo.NAModule4.description = Zusatz-Indoor-Modul, liefert Daten, wie z.B. Temperatur, Luftdruck, Luftfeuchtigkeit und CO2-Gehalt.
+
+thing-type.netatmo.NAPlug.label = Thermostat Relais
+thing-type.netatmo.NAPlug.description = Thermostat Relais.
+
+thing-type.netatmo.NATherm1.label = Heizk�rperthermostat
+thing-type.netatmo.NATherm1.description = Heizk�rperthermostat, dient zur Steuerung von Heizk�rpern und liefert Daten, wie z.B. Temperatur.
+
+# thing types config
+thing-type.config.netatmo.device.equipmentId.label = MAC-Adresse Modul
+thing-type.config.netatmo.device.equipmentId.description = MAC-Adresse des Haupt-Indoor-Moduls.
+
+thing-type.config.netatmo.device.refreshInterval.label = Abfrageintervall
+thing-type.config.netatmo.device.refreshInterval.description = Intervall zur Abfrage der Netatmo API (in Millisekunden).
+
+thing-type.config.netatmo.module.equipmentId.label = MAC-Adresse Zusatzmodule
+thing-type.config.netatmo.module.equipmentId.description = MAC-Adresse des Zusatzmoduls.
+
+thing-type.config.netatmo.module.parentId.label = MAC-Adresse Haupt-Indoor-Modul
+thing-type.config.netatmo.module.parentId.description = MAC-Adresse des Haupt-Indoor-Moduls.
+
+# channel types
+channel-type.netatmo.co2.label = CO2-Gehalt
+channel-type.netatmo.co2.description = Zeigt den aktuellen CO2-Gehalt (in ppm) an.
+
+channel-type.netatmo.temperature.label = Temperatur
+channel-type.netatmo.temperature.description = Zeigt die aktuelle Temperatur (in �C) an.
```

Comment:
Is this correct?
Does it show "°C" even when the user defined "°F" as the default unit?
Same goes for the other descriptions like pressure.